biscuit/src/Auth/Biscuit/Datalog/ScopedExecutor.hs

```diff
@@ -276,23 +278,18 @@ collectWorld blockId Block{..} =
       , FactGroup $ Map.singleton (Set.singleton blockId) $ Set.fromList bFacts
       )
 
--- | Query the facts generated by the authority and authorizer blocks
```

Comment:
Looks like you dropped a function and its documentation to replace it by two new ones without haddock. Was it intended?

Reply:
The function has been moved to `Biscuit.Token`, where it's properly documented, along with the new `queryRawBiscuitFacts`

The two new functions in this module are not meant to be used directly.